store/rootmulti/store.go

```diff
@@ -64,7 +66,9 @@ func NewStore(dir string, logger log.Logger) *Store {
 // Implements interface Committer
 func (rs *Store) Commit() types.CommitID {
 	var changeSets []*memiavl.NamedChangeSet
-	for key, store := range rs.stores {
+	for key := range rs.stores {
+		// it'll unwrap the inter-block cache
+		store := rs.GetCommitKVStore(key)
```

Comment:
do we need skip these 2 [types](https://github.com/cosmos/cosmos-sdk/blob/be081b10515fce26aac2dbfd9ea0f4a8052e0bd1/store/rootmulti/store.go#L1207) when commit?

Reply:
I think should call the commit method, but ignore the returned commit id